iphone/Classes/TiUIProgressBar.m

```diff
@@ -67,7 +67,10 @@ -(CGFloat)contentHeightForWidth:(CGFloat)width
 {
 	CGSize fontSize = [self sizeForFont:width];
 	CGSize progressSize = [progress sizeThatFits:fontSize];
-	return fontSize.height + progressSize.height;
+    if (messageLabel == nil) {
+        return fontSize.height + progressSize.height;
+    }
+	return fontSize.height + progressSize.height + 5;
```

Comment:
@hansemannn,  If you see function -(void)layoutSubviews in same file, there is code snippet -
	float fittingHeight = barSize.height + messageSize.height + 5;

which sets the gap of 5 pixel between message and progress bar. So while calculating height it should also give +5 pixel, if message is there.